### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by calebmackdavenport**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @jkhusanov @ChiefoftheOwls @marquessv
+* @jkhusanov @ChiefoftheOwls @marquessv @calebmackdavenport
```
Comment (jkhusanov) [inline .github/CODEOWNERS:1]: Could we remove `@ChiefoftheOwls @marquessv` ?

**Review by joshuakg — APPROVED** [review]

**Review by TannerDrayton — APPROVED** [review]

**Review by rkyle35242 — APPROVED** [review]
Do we need more folks than just you two?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1 @@
-* @jkhusanov @ChiefoftheOwls @marquessv
+* @jkhusanov @calebmackdavenport @rkyle35242
```
[the remark this answers is not in the document]
Comment (rkyle35242) [reply .github/CODEOWNERS:1]: jk, I'm cool to be on here

**Comment by rkyle35242** [comment]
@calebmackdavenport let's just get all the seniors+ on here

**Review by rkyle35242 — CHANGES_REQUESTED** [review]
Let's just add all the engineers that are Senior and up

**Review by jkhusanov — APPROVED** [review]

**Pull request merged**